### PR TITLE
Ozone Bid Adapter: Support schain module

### PR DESCRIPTION
**Pull request by afsheenb**
## Type of change
- [X] Feature

## Description of change
- Added support for client side sChain module

- PR for updated documentation can be found at https://github.com/prebid/prebid.github.io/pull/3461

engineering@ozoneproject.com

### CODE REVIEW

**Comment by afsheenb** [comment]
The circleCI error for the build of this PR is unrelated to our adapter:

```
1) builds and sends data
     finteza analytics adapter track bid response
     AssertionError: expected 'POST' to equal 'GET'
    at Context.<anonymous> (webpack:///test/spec/modules/fintezaAnalyticsAdapter_spec.js:122:46 <- test/test_index.js:322190:143
```

**Comment by patmmccann** [comment]
Can you fix your longstanding lgtm alert in this pr?

**Review by patmmccann — COMMENTED** [review]
plz fix https://lgtm.com/projects/g/prebid/Prebid.js/snapshot/c6ded29e3bcadcc3ec5ebc3cefb85400271d95cd/files/modules/ozoneBidAdapter.js?sort=name&dir=ASC&mode=heatmap#x225042b871758e91:1 line 603, useless assignment

**Comment by afsheenb** [comment]
Done @patmmccann, fix submitted for the unused assignment.

**Comment by afsheenb** [comment]
Again, the circleCI error is unrelated to our adapter changes:

```
1) should do nothing when bid metadata is not set
     FeedAdAdapter event tracking calls onTimeout handler
     AssertionError: expected 1 to equal 0
    at Context.<anonymous> (webpack:///test/spec/modules/feedadBidAdapter_spec.js:445:45 <- test/test_index.js:321704:142)
```

File: modules/ozoneBidAdapter.js [hunk 1]
```diff
@@ -298,33 +288,30 @@ export const spec = {
         }
       }
       if (fpd && deepAccess(fpd, 'site')) {
-        // attach the site fpd into exactly : imp[n].ext.[whitelabel].customData.0.targeting
-        logInfo('added FPD site object');
+        logInfo('added fdp.site');
```
Comment (patmmccann) [inline modules/ozoneBidAdapter.js:291]: typo

**Comment by patmmccann** [comment]
LGTM ; just one typo to fix plz

**Review by patmmccann — APPROVED** [review]

File: modules/ozoneBidAdapter.js [hunk 2]
```diff
@@ -4,16 +4,17 @@ import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes.js';
 import {config} from '../src/config.js';
 import {getPriceBucketString} from '../src/cpmBucketManager.js';
 import { Renderer } from '../src/Renderer.js';
+
```
Comment (patmmccann) [inline modules/ozoneBidAdapter.js:7]: this is making a linting error

**Review by patmmccann — CHANGES_REQUESTED** [review]
new linting error to fix, then appears good to go

**Review by patmmccann — APPROVED** [review]

**Pull request merged**